scripts/check_frontend_coverage.py

```diff
@@ -314,26 +301,26 @@ def check_coverage_changes():
         if file_name in whitelist:
             if total_lines != covered_lines:
                 errors += ('\033[1m{}\033[0m file is in the whitelist but its'
-                ' coverage decreased. Make sure it is fully covered'
-                ' by Karma unit tests.\n'.format(file_name))
+                           ' coverage decreased. Make sure it is fully covered'
+                           ' by Karma unit tests.\n'.format(file_name))
 
             whitelist.remove(file_name)
         else:
-            if total_lines == covered_lines:
+            if total_lines and total_lines == covered_lines:
```

Comment:
I'm a bit concerned about the change here. Why is this part checking whether total_lines is nonzero? Conceptually, I think files with zero total lines should be included in the whitelist (in case they expand in the future, we still want them to have 100% coverage) but I don't understand why we have files with zero total lines in the first place.

If it's possible for stanza.total_lines to be None, that seems like a more serious error that should be caught and dealt with earlier.

Reply:
This change was made because of sourceMap issue. I was facing some problems with it and I've putted this condition to understand what was happening. I forgot to remove it. We don't have any files with zero lines. However, I think it's totally make sense to include future files which may have zero lines in the whitelist.

The stanza.total_lines is already being checked in the __init__ method of LcovStanzaRelevantLines.